lighthouse-extension/app/src/lighthouse-ext-background.js

```diff
@@ -95,6 +95,9 @@ async function runLighthouseInExtension(flags, categoryIDs) {
  * @return {Promise<string|Array<string>|void>}
  */
 async function runLighthouseInLR(connection, url, flags, {lrDevice, categoryIDs, logAssets}) {
+  // Certain fixes need to kick-in under LR, see https://github.com/GoogleChrome/lighthouse/issues/5839
```

Comment:
One if the origin RTT estimates only applies in LR. Do you want to re use this there? Fine if you don't.